app/src/screens/StackNavigator.tsx

```diff
@@ -36,19 +32,15 @@ const StackNavigator = () => {
     loginAndRegister?.status === 'fulfilled';
   if (!isLoggedIn) {
     return (
-      <Stack.Navigator>
+      <Stack.Navigator screenOptions={{ headerShown: false }}>
         <Stack.Screen name="Sign In" component={SignIn} />
       </Stack.Navigator>
     );
   }
 
   return (
-    <Stack.Navigator>
-      <Stack.Screen
-        name="App"
-        options={{ headerShown: false }}
-        component={TabNavigator}
-      />
+    <Stack.Navigator screenOptions={{ headerShown: false }}>
```

Comment:
Does this change mean we can't navigate back to the app from a call or call invite screen, without using the device's controls?

Reply:
<img width="406" alt="Screenshot 2023-08-31 at 3 25 17 PM" src="https://github.com/twilio/twilio-voice-react-native-app/assets/35968892/31dea756-b0f4-408e-b55f-cc8cad669a3b">

I don't think so. I think this just prevents the header from being shown. I dont believe normal apps show headers on every screen.